versioned/tiered/rocksdb/src/main/java/com/dremio/nessie/versioned/store/rocksdb/RocksDBPathVisitor.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.dremio.nessie.versioned.store.rocksdb;
+
+import com.dremio.nessie.versioned.impl.condition.ExpressionPath;
+import com.google.common.base.Preconditions;
+
+/**
+ * Visitor for creating String path representations from ExpressionPaths.
+ */
+class RocksDBPathVisitor implements ExpressionPath.PathVisitor<Boolean, String, RuntimeException> {
```

Comment:
This seems like a common pattern to need to convert to String - should it be moved into ExpressionPath as a default visitor?

Reply:
This is refactored now to store ExpressionPath in Function. I then drill down into that as needed in the evaluate method.
This visitor class is no longer needed.